docs/cpp_client/cpp_quick_start.md

```diff
@@ -20,52 +17,209 @@ Ubuntu 18.04.
+    - [CMake Tools](https://marketplace.visualstudio.com/items?itemName=ms-vscode.cmake-tools)
+    - [C/C++](https://marketplace.visualstudio.com/items?itemName=ms-vscode.cpptools)
 
-    1. Click the Configure "All Projects" icon: 
 
-        <p class="img_container">
-        <img class="sm_img" src="../imgs/configure.png">
-        </p>
+3. Select `File -> Open Folder` and navigate to this folder to build the cpp-examples or simulator-cpp-client to build just the client library.
+4. Use the CMake extension to configure and build
+    1. Click the Configure All Projects icon:
 
-    1. If prompted to Scan for Kits select "Yes":
+    <div class="img_container">
+        <img class="lg_img" src="../imgs/configure.png">
+    </div>
 
-        **Windows**: `Visual Studio Community 2019 Release - amd64`
-    
-        **Linux**: Choose the compiler of your choice, tested with `g++ 7.5.0`
-    
-    1. Build the client by clicking the `Build All Projects` icon:
-    
-        <p class="img_container">
-        <img class="sm_img" src="../imgs/build.png">
-        </p>
+    2. If prompted to Scan for Kits select Yes.
+
+        **Windows**: Use `Visual Studio Community 2019 Release - amd64`.
+
+        **Linux**: Use the compiler of your choice. Tested with `g++ 7.5.0`.
+
+    3. Build the client by clicking the `Build All Projects` icon:
+
+    <div class="img_container">
+        <img class="lg_img" src="../imgs/build.1.png">
+    </div>
+
+### Set Launch Target
+
+<div class="img_container">
+    <img class="lg_img" src="../imgs/dev_target_set_example_v3.jpeg">
+</div>
+
+**NOTE**: Launching automatically fires off a build check in cmake so just setting the launch target will suffice.
+
+
+### Run Example
+
+#### Windows
+
+<div class="img_container">
+    <img class="wide_img" src="../imgs/example_windows_vscode.jpeg">
+</div>
+
+*Example configuration with `fisheye_camera_equidistant` as a build + launch target.*
+
+After you've set your build configuration and target, you can run by hitting `F5` to run in debug or `Ctrl+F5` to run without debugging.
+
+## Installation with CMake
+You can include the monoDrive Simulator client in your existing CMake project
+by adding the following lines to your `CMakeLists.txt`
+
+```cmake
+# add monodrive client library from local repo
+add_subdirectory(path/to/monodrive-client/monodrive mdclient)
+
+# link targets as needed
+target_link_libraries(<mytarget> monodrive)
+```
+
+
+## Installation with Bazel
+You can include the monoDrive Simulator client in your existing Bazel project
+by adding the following lines to your `WORKSPACE` file
+```
+local_repository(
+    name = "monodrive",
+    path = "path/to/monodrive-client"
+)
+```
+and the following to your `BUILD` files as needed
+```
+cc_library(
+    name = "my_lib",
+    srcs = [...],
+    hdrs = [...],
+    deps = [
+        ...,
+        "@monodrive//monodrive/core:monodrive"
+    ]
+)
+```
+
+
+## Installation to system on Ubuntu
```

Comment:
As far as I know, there is no `install` target for the client. I think this should be removed.

Reply:
Are these the instructions for installation with Bazel?